src/references.js

```diff
@@ -170,16 +170,16 @@ export default function findReferences(cell, globals) {
         identifier(node, parents);
         break;
       }
-      case "ArrayPattern":
+      case "ArrayPattern": {
```

Comment:
minor nit (feel free to ignore) -- now that the `identifier` function is only used in one `case`, is it worth inlining it? That would remove the shadowing of the `identifier` function, as well as remove the unnecessary mix of `parents` and `nodeParents` (which are the same) in that function.